features/SHA_Extractor/sha_extractor.feature

```diff
@@ -1,104 +1,105 @@
-      And Kafka topic specified in configuration variable "archive_results" is created
-     When new event about about new tarball being sent by Insights Operator is produced into "incoming_topic" topic
-     Then SHA extractor should consume message about this event
-      And this message should contains following attributes
-          | Attribute    | Description                | Type         |
-          | account      | account ID                 | unsigned int |
-          | principal    | principal ID               | unsigned int |
-          | size         | tarball size               | unsigned int |
-          | url          | URL to S3                  | string       |
-          | b64_identity | identity encoded by BASE64 | string       |
-          | timestamp    | timestamp of event         | string       |
-     When SHA extractor consume this message
-     Then SHA extractor should take the URL attribute
-      And SHA extractor should download tarball from given URL attribute
-     When SHA extractor decompress the tarball
-     Then SHA extractor needs to check if it contains file "config/workload_info.json"
-     When that file is not found
-     Then the tarball is not further processed
-
-
-  Scenario: Check if SHA extractor is able to finish the processing of SHA images
-    Given SHA extractor service is started
-      And Kafka broker is started
-      And Kafka topic specified in configuration variable "incoming_topic" is created
-      And Kafka topic specified in configuration variable "dead_letter_queue_topic" is created
-      And Kafka topic specified in configuration variable "archive_results" is created
-     When new event about about new tarball being sent by Insights Operator is produced into "incoming_topic" topic
-     Then SHA extractor should consume message about this event
-      And this message should contains following attributes
-          | Attribute    | Description                | Type         |
-          | account      | account ID                 | unsigned int |
-          | principal    | principal ID               | unsigned int |
-          | size         | tarball size               | unsigned int |
-          | url          | URL to S3                  | string       |
-          | b64_identity | identity encoded by BASE64 | string       |
-          | timestamp    | timestamp of event         | string       |
-     When SHA extractor consume this message
-     Then SHA extractor should take the URL attribute
-      And SHA extractor should download tarball from given URL attribute
-     When SHA extractor decompress the tarball
-     Then SHA extractor needs to check if it contains file "config/workload_info.json"
-     When that file is found
-     Then the content of this file needs to be sent into topic "archive_results"
+#  Scenario: Check if SHA extractor is able to consume messages from Kafka
```

Comment:
is this a leftover?

Reply:
no, i will uncomment those when the code is ready, right now I implemented only one scenario